lib/kubernetes-deploy/kubernetes_resource/job.rb

```diff
@@ -30,8 +31,20 @@ def status
       end
     end
 
+    def failure_message
+      if (condition = failed_status_condition.presence)
```

Comment:
Why the parentheses?

Reply:
Its a habit from a style checker, but basically saying I'm intentionally using `=`  and didn't mean to use `==`